### ISSUE TITLE
showcase: no images displayed in deployed showcase on ghPages due to wrong URL

### ISSUE THREAD

**Issue by christophercr** [NationalBankBelgium/stark#1036]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#got-a-question-or-problem
</code></pre>

## Current behavior

The latest version of the Showcase deployed in ghPages ([version 10.0.0-beta.2](https://stark.nbb.be/showcase/10.0.0-beta.2/)) is not working properly due to the following errors:

```
GET https://showcase/10.0.0-beta.2//showcase/10.0.0-beta.2/assets/images/app-header.png net::ERR_NAME_NOT_RESOLVED
GET https://showcase/10.0.0-beta.2//showcase/10.0.0-beta.2/assets/stark-core/logo/stark_logo_transparent.svg net::ERR_NAME_NOT_RESOLVED
```

The different images on assets cannot be loaded due to the duplication of the deploy URL part: `showcase/10.0.0-beta.2//showcase/10.0.0-beta.2`.

Apart from this, there are some assets that cannot be loaded because the URL contains the placeholders `<stark-dummy-base-href>` and `<stark-dummy-deploy-url>` that were not correctly replaced when deploying the app to GhPages.

## Expected behavior

The Showcase app should be fully working in all possible deployUrl's: `latest` and specific version.

## Minimal reproduction of the problem with instructions

1. Open the Showcase app in https://stark.nbb.be/showcase/10.0.0-beta.2/.
2. After the application has initialized, open the browser console. The console displays the following errors:

```
GET https://showcase/10.0.0-beta.2//showcase/10.0.0-beta.2/assets/images/app-header.png net::ERR_NAME_NOT_RESOLVED
GET https://showcase/10.0.0-beta.2//showcase/10.0.0-beta.2/assets/stark-core/logo/stark_logo_transparent.svg net::ERR_NAME_NOT_RESOLVED
```

## Environment

<pre><code>
Angular version: 7.1.1
Stark version: 10.0.0-beta.2

</code></pre>

**Comment by christophercr** [comment]
This issue should be solved once #883 is implemented.

**Comment by christophercr** [comment]
Solved in #1148
